aequilibrae/matrix/aequilibrae_matrix.py

```diff
@@ -171,7 +182,9 @@ def create_empty(
                                 "than {}: {}".format(INDEX_NAME_MAX_LENGTH, ind_name)
                             )
                     else:
```

Comment:
Try moving your raise to the true case... then you don't need an else case at all

Reply:
Done